include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -23,6 +23,10 @@
 #include "sycl_defs.h"
 
 #include <type_traits>
+#include <utility> // std::pair
```

Comment:
Is these comments really required?

Reply:
No, these are not required, but I personally find it quite useful to explain why this or that standard header is included, especially when you need only a thing or two.